### PR TITLE
Fix tests for both jwt ^3.4.3 and ^4 (^4.0.0 and ^4.1.0))

### PR DESCRIPTION
**Pull request by Richard87**
All tests are green for all current jwt versions on ^3.4.3 and ^4

### CODE REVIEW

**Comment by Richard87** [comment]
Fixes #34

File: tests/ResponseTypes/IdTokenResponseTest.php [hunk 1]
```diff
@@ -86,30 +79,38 @@ public function testThrowsRuntimeExceptionWhenMissingIdentifierSetInterface()
...
+        self::assertObjectHasAttribute('refresh_token', $json);
+        self::assertObjectHasAttribute('id_token', $json);
+
+        if (class_exists("\Lcobucci\JWT\Token\Parser")) {
+            $parser = new \Lcobucci\JWT\Token\Parser(new \Lcobucci\JWT\Encoding\JoseEncoder);
+        } else {
+            $parser = new \Lcobucci\JWT\Parser();
+        }
+
+        var_dump($json->id_token);
```
Comment (tjveldhuizen) [inline tests/ResponseTypes/IdTokenResponseTest.php:111]: This `var_dump()` should not be here, I assume

**Review by tjveldhuizen — COMMENTED** [review]
I have tested your code in my project and it looks good. It would be nice if it could be merged and released after you've removed the var_dump.

**Comment by steverhoades** [comment]
Thanks @Richard87 as soon as you remove the `var_dump` I'll get this merged and tagged for immediate release.

**Comment by Richard87** [comment]
Oops :O 

Thanks! I assume we can delete the class_exist related code in a future major version bump, where you can require only Jwt 4 :+1:

**Comment by Richard87** [comment]
Also, I added PHP8 and JWT4 to travis' test matrix, I hope that's okay, and also, it's the first time I have ever checked out travis, so I appreciate if you would take a look and see if it's okay :)

**Pull request merged**